.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,6 +26,14 @@ assignees:
 
 ```
 
+##### COLLECTION VERSION
+<!--- Paste verbatim output from "ansible-galaxy collection list | grep <collection>"  between the quotes
+for example: ansible-galaxy collection list | grep community.general
```

Comment:
```suggestion
<!--- Paste verbatim output from "ansible-galaxy collection list <namespace>.<collection>"  between the quotes
for example: ansible-galaxy collection list community.general
```

No grep required, I learned it last week -- for example:
```
> ansible-galaxy collection list community.general

# /home/dmsimard/dev/virtualenvs/ansible/lib/python3.9/site-packages/ansible_collections
Collection        Version
----------------- -------
community.general 2.0.1  

# /home/dmsimard/dev/virtualenvs/ansible/lib64/python3.9/site-packages/ansible_collections
Collection        Version
----------------- -------
community.general 2.0.1  
```

Reply:
@dmsimard Great, that's even better! I thought that `list` would simply list all collections. Didn't know you can specify a collection there. Thanks :-)